Resources/config/runner.xml

```diff
@@ -9,6 +9,6 @@
     </parameters>
 
     <services>
-        <service id="liip_monitor.runner" class="%liip_monitor.runner.class%" />
+        <service id="liip_monitor.runner" class="%liip_monitor.runner.class%" public="true" />
```

Comment:
this service should actually not be public since its an internal service ..
we just need it to be accessible during tests :-/

Reply:
then make it so on the tests only, but i see it still fails on your PR

Reply:
hmm .. actually we do use this service in the CLI command